AndelaOSP/Andela-Socials
# #167051504 Document frontend Methods

#### What Does This PR Do?
It adds documentation(JSDoc) to the frontend codebase

#### Description Of Task To Be Completed
- [x] add JSDOc to the existing frontend methods/functions
- [x] refactor some of the existing frontend codebase

#### Any Background Context You Want To Provide?
Documenting the frontend codebase is quite large so the task has been broken down into two parts and this is the first part of the task

#### How can this be manually tested?
- Fetch the branch locally by running `git fetch origin ch-document-frontend-1-167051504` & `git checkout  ch-document-frontend-1-167051504` respectively
- Check through the codebase to see the documentation

#### What are the relevant pivotal tracker stories?
[#167051504](https://www.pivotaltracker.com/story/show/167051504)

#### Screenshot(s)
N/A
